doc/about.rst

```diff
@@ -1,8 +1,11 @@
-About Panel
-===========
+About
+=====
 
-Panel is completely open source, available under a BSD license freely for both commercial and non-commercial use. Panel is part of the HoloViz ecosystem and works well with all the HoloViz tools.
+Panel is completely open source, available under a BSD license freely for both commercial and non-commercial use.
 
-Panel was originally developed with the support of `Anaconda Inc. <https://anaconda.com>`_, and is now maintained by Anaconda developers and community contributors.
+Panel is part of the `HoloViz <https://holoviz.org>`_ family of tools. The `holoviz.org <https://holoviz.org>`_ website shows how to use Panel together with other libraries to solve complex problems, with detailed tutorials and examples. Example projects using Panel are at
+`examples.pyviz.org <https://examples.pyviz.org>`_, and you can compare Panel to other available tools at `pyviz.org <https://pyviz.org>`_.
 
-If you are interested in contributing to Panel development, you can see our list of open issues that you could help address, including possible larger wishlist items. And if you are lucky enough to be in a position to fund our developers to work on documentation, features, or bug fixes, please contact sales@anaconda.com.
+If you have any questions or usage issues visit the `HoloViz Discourse <https://discourse.holoviz.org/c/panel/`_. If you are interested in contributing to Panel development to help address some of the `open issues <https://github.com/holoviz/panel/issues>`_, see our `developer instructions <https://pyviz-dev.github.io/panel/developer_guide/index.html>`_ to set up your development environment.
```

Comment:
```suggestion
If you have any questions or usage issues visit the `Panel Discourse <https://discourse.holoviz.org/c/panel/`_ site. If you are interested in contributing to Panel development to help address some of the `open issues <https://github.com/holoviz/panel/issues>`_, see our `developer instructions <https://pyviz-dev.github.io/panel/developer_guide/index.html>`_ to set up your development environment.
```

(The link goes straight to the Panel section; I don't _think_ it's important to mention that it's part of the HoloViz Discourse site.)